### PR TITLE
rgw: support radosgw-admin zone/zonegroup placement get command

### PR DESCRIPTION
**Pull request by jiahuizeng**
https://tracker.ceph.com/issues/42291

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by jiahuizeng** [comment]
jenkins test make check

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -478,6 +480,7 @@ enum {
   OPT_ZONEGROUP_PLACEMENT_MODIFY,
   OPT_ZONEGROUP_PLACEMENT_RM,
   OPT_ZONEGROUP_PLACEMENT_LIST,
+  OPT_ZONEGROUP_PLACEMENT_GET,
```
Comment (cbodley) [inline src/rgw/rgw_admin.cc]: can you please add OPT_ZONEGROUP_PLACEMENT_GET and OPT_ZONE_PLACEMENT_GET to both of `raw_storage_ops_list` and `readonly_ops_list`?

Comment (jiahuizeng) [reply src/rgw/rgw_admin.cc]: Thank you for your suggestion. OPT_ZONEGROUP_PLACEMENT_GET and OPT_ZONE_PLACEMENT_GET have been added to raw_storage_ops_list and readonly_ops_list.

**Comment by jiahuizeng** [comment]
jenkins test make check

**Comment by cbodley** [comment]
jenkins test make check

**Comment by liuchang0812** [comment]
ping

**Comment by cbodley** [comment]
passed qa in http://pulpito.ceph.com/cbodley-2020-01-23_17:19:37-rgw-wip-cbodley-testing-distro-basic-smithi/

**Comment by smithfarm** [comment]
What about `doc/man/8/radosgw-admin.rst` ?

**Pull request merged**